kit/src/lib/FrameworkContentBlock.svelte

```diff
@@ -0,0 +1,77 @@
+<script lang="ts">
+    import type { SvelteComponent } from "svelte";
+    import type { Framework } from "./types";
+
+    import { onMount} from "svelte";
+    import { getFrameworkStore } from "./stores";
+	import IconPytorch from "./IconPytorch.svelte";
+	import IconTensorflow from "./IconTensorflow.svelte";
+	import IconJax from "./IconJax.svelte";
+	import IconEyeShow from "./IconEyeShow.svelte";
+	import IconEyeHide from "./IconEyeHide.svelte";
+
+    export let framework: Framework;
+
+    const FRAMEWORK_CONFIG: Record<Framework, {Icon: typeof SvelteComponent, label: string}> = {
+		"pytorch": {
+            Icon: IconPytorch,
+            label: "Pytorch",
+        },
+		"tensorflow": {
+            Icon: IconTensorflow,
+            label: "TensorFlow",
+        },
+		"jax": {
+            Icon: IconJax,
+            label: "JAX",
+        },
+	};
+    const { Icon, label } = FRAMEWORK_CONFIG[framework];
+    const localStorageKey = `hf_doc_framework_${framework}_is_hidden`;
+    const frameworkStore = getFrameworkStore(framework);
+
+    function toggleHidden(){
+        $frameworkStore = !$frameworkStore;
+        localStorage.setItem(localStorageKey, $frameworkStore ? "true" : "false");
+    }
+
+    onMount(() => {
+        if(localStorage.getItem(localStorageKey) === "true"){
```

Comment:
using [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) for persistance over multiple sessions for a user. wdy

Reply:
looks good!